bm-persona/Libraries/LibraryTableViewCell.swift

```diff
@@ -8,9 +8,22 @@
 
 import UIKit
 
-class MyCell: UITableViewCell {
-
-
+class LibraryTableViewCell: UITableViewCell {
+    
+    static let kCellIdentifier = "libraryCell"
+    
+    override var frame: CGRect {
+        get {
+            return super.frame
+        }
+        set (newFrame) {
+            var frame = newFrame
+            frame.origin.x += 5
+            frame.size.width -= 10
+            super.frame = frame
+        }
+    }
+    
     override func layoutSubviews() {
         super.layoutSubviews()
         contentView.frame = contentView.frame.inset(by: UIEdgeInsets(top:0, left:0, bottom:10, right:0))
```

Comment:
Why not remove the override var and set the left / right edge insets using frame.inset?